doc/internal/man3/ossl_provider_new.pod

```diff
@@ -92,6 +94,9 @@ one was loaded, and the I<provider object> will be freed.
 ossl_provider_add_module_location() adds a location to look for a
 provider module.
 
+ossl_provider_set_fallback() marks an available provider as fallback.
+If the given provider is already "activated", this has no effect.
```

Comment:
Presumably it will remain a fallback in the event that it is subsequently unloaded, so it still has an effect in that case.

Reply:
True, but that also requires that *all* activated providers are deactivated at that point.

Reply:
Right...its a minor point. Still approved.